src/renderer/components/table/table.tsx

```diff
@@ -62,6 +64,29 @@ export class Table extends React.Component<TableProps> {
     sortSyncWithUrl: true,
   };
 
+  constructor(props: TableProps) {
+    super(props);
+
+    if (this.props.sortable) {
+      const storage = createStorage<Record<string, Partial<TableSortParams>>>("table_sort_params", {});
```

Comment:
I think that the storage should probably be created unconditionally. Please refactor this to be more like how #2279 does it.